_posts/august-2024-newsletter.md

```diff
@@ -0,0 +1,60 @@
+---
+title: "August Newsletter"
+excerpt: "Our August newsletter highlights key events, including our General Meeting on August 18 and upcoming public meetings at the DART Board and Dallas City Council."
+coverImage: "/assets/blog/august-2024-newsletter/cover.jpg"
+date: "2024-08-09T05:00:00.000Z"
+author:
+  name: DATA Newsletter
+  picture: "/assets/blog/authors/data-team.png"
+ogImage:
+  url: "/assets/blog/august-2024-newsletter/cover.jpg"
+---
+
+We have three upcoming events that I want you to be aware of this month.  
+
+## August General Meeting
+
+---
+First, our August General Meeting will be on August 18, 2024 from 3pm to 5pm at the Metropolitan [1200 Main Street, Dallas](https://maps.app.goo.gl/pPBSUsQeJYZr1hxf7) (Enter via Murphy Drive) on the 6th floor, same room as the first meeting we held. We will have light snacks and refreshments available.
+
+We will be discussing our strategy for two upcoming public meetings that we want to show up in large numbers for.
+
+## DART Board Meeting
+
+---
```

Comment:
Horizontal lines shouldn't be used here or anywhere else, probably

Reply:
was imported from the email but we can remove it if it doesn't look good